package.json

```diff
@@ -123,8 +123,8 @@
     "serve-static": "^1.15.0"
   },
   "engines": {
-    "node": ">=16.13.0",
-    "npm": ">=8.0.0"
+    "node": ">=20.14.0",
```

Comment:
`balena-api` and `open-balena-api` are at `^22.2.0`, whats the reason for staying at v20 here?

Reply:
Because pinejs is a library that could be used by other things as well so we usually stick to at least supporting the LTS release. If there was a feature from the current release that we really needed then we could pin to that but otherwise sticking to LTS makes sense